src/app/containers/CpsAssetMediaPlayer/index.jsx

```diff
@@ -78,11 +83,15 @@ CpsAssetMediaPlayer.propTypes = {
   assetUri: string.isRequired,
   isLegacyMedia: bool,
   showLoadingImage: bool,
+  bottomPadding: bool,
+  showCaption: bool,
```

Comment:
All the other `bool` props are named in a way that makes it obvious they're bool props - do you think it makes sense for `bottomPadding` to follow suite?

Reply:
I can make this change 👍